ImageTransform/CommonPart/transform/Transformation.hpp

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include "ColorCalc.hpp"
+#include "Constants.hpp"
+#include <QPoint>
+#include <QPointF>
+#include <QPolygon>
+#include <QPolygonF>
+#include <QRect>
+#include <QRectF>
+#include <cmath>
+#include <complex>
+#include <iostream>
+
+namespace common
+{
+
+template <typename T_dig, typename T_point, typename T_rect, typename T_poly>
+class Transform
+{
+
+public:
+    Transform(const T_point& rotCenter, T_dig deg_angle)
+        : m_rotCenter{rotCenter}, m_angle{deg_angle}
+    {
+    }
+
+    T_point operator()(const T_point& point)
```

Comment:
This method should be `const` and `m_angle` should not be changed.

Reply:
Corrected.